src/libs/ReportActionComposeFocusManager.ts

```diff
@@ -29,7 +29,7 @@ function onComposerFocus(callback: FocusCallback, isMainComposer = false) {
 /**
  * Request focus on the ReportActionComposer
  */
-function focus() {
+function focus(shouldFocusForNative?: boolean) {
```

Comment:
Having a `shouldFocusForNative` param seems to encourages code that is not cross platform.

Can we rename this to indicate the intended behavior regardless of what platform we are on?

Reply:
>Can we rename this to indicate the intended behavior regardless of what platform we are on?

That isn't clear @marcaaron. Because we are early returning here
https://github.com/Expensify/App/blob/a4d01a87b242eef06af37db965bd135745e342e5/src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.tsx#L612-L613
`willBlurTextInputOnTapOutside` is false only for natives so we are normally preventing focus for natives but in our case we have to manually focus it for natives too that's why I came up with the variable. What other name could we give?

Reply:
Naming it `shouldFocus` should be fine @FitseTLT. We do not support code that this written just for native platforms mixed together with the rest of the code. If we need to do something specific to the native platform, we usually create a separate file `.native.ts` extension to keep proper separation of concerns.